src/main/java/uk/gov/hmcts/divorce/caseworker/schedule/CaseworkerIssueAosTask.java

```diff
@@ -0,0 +1,40 @@
+package uk.gov.hmcts.divorce.caseworker.schedule;
+
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.scheduling.annotation.Scheduled;
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.divorce.caseworker.service.CcdManagementException;
+import uk.gov.hmcts.divorce.caseworker.service.CcdSearchService;
+import uk.gov.hmcts.divorce.caseworker.service.CcdUpdateService;
+import uk.gov.hmcts.reform.ccd.client.model.CaseDetails;
+
+import java.util.List;
+
+import static uk.gov.hmcts.divorce.caseworker.event.CaseworkerIssueAos.CASEWORKER_ISSUE_AOS;
+import static uk.gov.hmcts.divorce.common.model.State.Issued;
+
+@Component
+@Slf4j
+public class CaseworkerIssueAosTask {
+
+    @Autowired
+    private CcdUpdateService ccdUpdateService;
+
+    @Autowired
+    private CcdSearchService ccdSearchService;
+
+    @Scheduled(cron = "${schedule.issue_aos}")
+    public void issueAosTask() {
+
+        log.info("Issue AOS Scheduled Task started.");
+
+        try {
+            final List<CaseDetails> issuedCases = ccdSearchService.searchForAllCasesWithStateOf(Issued);
+            issuedCases.forEach(caseDetails -> ccdUpdateService.submitEvent(caseDetails, CASEWORKER_ISSUE_AOS));
```

Comment:
If processing of one case fails then ideally we want other cases to be processed ?

Reply:
I failed fast here so that one of the nodes would not continue processing, but there is the possibility that it was a different failure, so maybe this should be finer grain in the exception types produced.

Reply:
It is not good to rely on nodes as sometimes only one nodes is functional so our job may not run at all unless we plan to run it at frequent  intervals for e.g every hour between 12-5 am